framework/src/base/ScalarCoupleable.C

```diff
@@ -84,6 +84,15 @@ ScalarCoupleable::coupledScalar(const std::string & var_name, unsigned int comp)
   return getScalarVar(var_name, comp)->number();
 }
 
+Order
+ScalarCoupleable::coupledScalarOrder(const std::string & var_name, unsigned int comp)
+{
+  if (!isCoupledScalar(var_name, comp))
+    return FIRST;
```

Comment:
But default coupling in not always on, see [addCoupledVar(name, doc_string)](https://github.com/idaholab/moose/blob/devel/framework/include/utils/InputParameters.h#L267)